components/Dashboard/Post/PostComments.tsx

```diff
@@ -71,7 +71,12 @@ const PostComments: React.FC<PostCommentsProps> = ({
               onChange={(e) => setPostCommentBody(e.target.value)}
               disabled={loading}
             />
-            <Button type="submit" disabled={loading} className="submit-btn">
+            <Button
+              type="submit"
+              disabled={loading}
```

Comment:
Same as previous comment
```suggestion
              loading={loading}
```

Reply:
🙌🏼 ✅